config/docker/chromeos-flash/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN echo "nameserver 1.1.1.1" > /etc/resolv.conf && \
```

Comment:
I don't think we need this hack in `resolv.conf`, the other Dockerfiles don't need it and build fine.

I did have to use that at some point on a local setup due to some conflicts so maybe you copied it from a sample Dockerfile?

Reply:
Removed the hack